utils/testdata/interfaceTestNoName.sol

```diff
@@ -0,0 +1,4 @@
+interface Interface {
+     function guess(uint8 n, address x) payable external;
+     function isComplete() view external returns (bool example, string );
```

Comment:
There's still a lingering space at the end which is kinda annoying, so if we can fix the parser to not emit that would be great, nbd otherwise